docs/modules/ROOT/pages/how-tos/metadata/attestations.adoc

```diff
@@ -130,13 +170,16 @@ Example output:
 📦 Supply Chain Security Related artifacts for an image: quay.io/redhat-user-workloads/rhn-support-csears-tenant/demo-build/partner-catalog-build-ucmg@sha256::<output omitted>
 └── 💾 Attestations for an image tag: quay.io/redhat-user-workloads/rhn-support-csears-tenant/demo-build/partner-catalog-build-ucmg:sha256-:<output omitted>.att
    ├── 🍒 sha256::<output omitted>
-   └── 🍒 sha256::<output omitted>
 └── 🔐 Signatures for an image tag: quay.io/redhat-user-workloads/rhn-support-csears-tenant/demo-build/partner-catalog-build-ucmg:sha256-:<output omitted>.sig
  └── 🍒 sha256::<output omitted>
 └── 📦 SBOMs for an image tag: quay.io/redhat-user-workloads/rhn-support-csears-tenant/demo-build/partner-catalog-build-ucmg:sha256-:<output omitted>.sbom
   └── 🍒 sha256:<output omitted>
 --
 
+== Inspecting the downloaded provenance
+
+The generated provenance includes information from the Tekton PipelineRun that generated the attested artifact. It 
```

Comment:
This sentence is a little hard to follow. I actually think if you just delete the first "generated," it would help. Also I think the "It" at the end is a typo.

Reply:
I think that this entire line was mistakenly included as well as the procedure above.